test/games/test_log_file_export.py

```diff
@@ -124,6 +124,8 @@ def pytest_generate_tests(metafunc: _pytest.python.Metafunc) -> None:
         "prime1_crazy_seed.rdvgame",  # chaos features
         "prime1_crazy_seed_one_way_door.rdvgame",  # same as above but 1-way doors
         "prime1_refills.rdvgame",  # Refill items + custom artifact count
+        # Prime 2
+        "prime2_starts_with_cannon_ball",
```

Comment:
missing .rdvgame at the end

(and the `prime2/` prefix given the other comment)